lib/client/websocket-tracker.js

```diff
@@ -133,6 +148,18 @@ WebSocketTracker.prototype._onSocketData = function (data) {
     return
   }
 
+  if (data.action === common.ACTIONS.ANNOUNCE || data.offer || data.answer) {
```

Comment:
This is defined in `common-node.js`, so it is excluded from browser builds by this line in `package.json`: https://github.com/feross/bittorrent-tracker/blob/7daf7ec2974f6a8de6b25ca01f75f8db82ce7f3e/package.json#L14 There should have been better code comments to explain this.

We should just use the string `'announce'`, especially because that's what that server actually checks for!